clang/test/CXX/drs/dr23xx.cpp

```diff
@@ -252,4 +252,14 @@ namespace dr2397 { // dr2397: 17
     auto (*c)[5] = &a;
   }
 } // namespace dr2397
+
+// CWG2363 was closed as NAD, but its resolution does affirm that
```

Comment:
I find it totally normal for DR tests to tests NAD issues to affirm the status quo. You don't have to put this comment if you don't really want to.